packages/ui/v2/core/Shell.tsx

```diff
@@ -742,59 +738,59 @@ function SideBarContainer() {
+                <Icon.FiArrowLeft className="h-4 w-4 flex-shrink-0 text-neutral-500 group-hover:text-neutral-900" />
+              </button>
+              <button
+                color="minimal"
+                onClick={() => window.history.forward()}
+                className="desktop-only group flex text-sm font-medium text-neutral-500 hover:text-neutral-900">
+                <Icon.FiArrowRight className="h-4 w-4 flex-shrink-0 text-neutral-500 group-hover:text-neutral-900" />
+              </button>
+              <KBarTrigger />
+            </div>
+          </header>
+
+          <hr className="desktop-only absolute -left-3 -right-3 mt-4 block w-full border-gray-200" />
+
+          {/* logo icon for tablet */}
           <Link href="/event-types">
-            <a className="px-4">
-              <Logo small />
+            <a className="text-center md:inline lg:hidden">
+              <Logo small icon />
             </a>
           </Link>
-          <div className="flex space-x-2">
-            <button
-              color="minimal"
-              onClick={() => window.history.back()}
-              className="desktop-only group flex text-sm font-medium text-neutral-500 hover:text-neutral-900">
-              <Icon.FiArrowLeft className="h-4 w-4 flex-shrink-0 text-neutral-500 group-hover:text-neutral-900" />
-            </button>
-            <button
-              color="minimal"
-              onClick={() => window.history.forward()}
-              className="desktop-only group flex text-sm font-medium text-neutral-500 hover:text-neutral-900">
-              <Icon.FiArrowRight className="h-4 w-4 flex-shrink-0 text-neutral-500 group-hover:text-neutral-900" />
-            </button>
-            <KBarTrigger />
-          </div>
-        </header>
 
-        <hr className="desktop-only absolute -left-3 -right-3 mt-4 block w-full border-gray-200" />
-
-        {/* logo icon for tablet */}
-        <Link href="/event-types">
-          <a className="text-center md:inline lg:hidden">
-            <Logo small icon />
-          </a>
-        </Link>
+          <Navigation />
+        </div>
 
-        <Navigation />
-      </div>
+        {isCalcom && <Tips />}
 
-      {isCalcom && <Tips />}
-      {/* Save it for next preview version
-       <div className="hidden mb-4 lg:block">
-        <UserV2OptInBanner />
-      </div> */}
-
-      <div data-testid="user-dropdown-trigger">
-        <span className="hidden lg:inline">
-          <UserDropdown />
-        </span>
-        <span className="hidden md:inline lg:hidden">
-          <UserDropdown small />
-        </span>
-      </div>
-      <DeploymentInfo />
-    </aside>
+        <div className="fixed left-1 bottom-0 w-4 bg-gray-50 before:absolute before:left-0 before:-top-20 before:h-20 before:w-48 before:bg-gradient-to-t before:from-gray-50 before:to-transparent lg:left-4 lg:w-48">
```

Comment:
Again, little change, besides the user dropdown + deployment info wrapped inside this fixed positioned div. The content of these divs hasn't changed.